packages/portal/src/components/embed/EmbedGateway.vue

```diff
@@ -127,7 +142,30 @@
     },
 
     mounted() {
-      this.provider = serviceForUrl(this.url);
+      this.providerUrl = this.url;
+
+      if (this.embedCode) {
+        const template = document.createElement('div');
+        template.innerHTML = this.embedCode;
+        const iframe = template.querySelector('iframe');
+        const script = template.querySelector('script');
```

Comment:
I'm doubting this is the best way to get the `src` and iframe dimensions from the embed code. This requires it to always run on the client (for document to be available). Also I'm not 100% sure if this is save as it does create iframe and/or script elements (although not appended to the body). Perhaps we could extract the `src`, `height` and `width` using a regex. Then we could call all this in `fetch`.